### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
🙂

### CODE REVIEW

File: data/translations/kdiskmark_nl_NL.ts [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+    </message>
+    <message>
+        <location filename="../../src/about.ui" line="146"/>
+        <source>E-mail:</source>
+        <translation>E-mailadres:</translation>
+    </message>
+    <message>
+        <location filename="../../src/about.ui" line="178"/>
+        <source>Flexible I/O Tester:</source>
+        <translation>Flexibele I/O-tester:</translation>
```
Comment (JonMagon) [inline data/translations/kdiskmark_nl_NL.ts:34]: This is the name of the package. It shouldn't be translated. It is possible to translate it for languages which may have differences like spaces before colons.

File: data/dev.jonmagon.kdiskmark.helper.policy [hunk 2]
```diff
@@ -13,6 +13,7 @@
       <message xml:lang="fr">Autorisation est requise pour mesurer la performance des disques</message>
       <message xml:lang="hu">Hitelesítés szükséges a lemezek teljesítményértékeléséhez</message>
       <message xml:lang="ja">特権的な KDiskMark の実行ヘルパー</message>
+      <message xml:lang="de">Authenticatie vereist om schijven te kunnen benchmarken</message>
```
Comment (JonMagon) [inline data/dev.jonmagon.kdiskmark.helper.policy:16]: de is for Deutsch

Comment (Vistaus) [reply data/dev.jonmagon.kdiskmark.helper.policy]: Sorry, my bad. I copied over the entire string to make sure the formatting was kept, but I accidentally forgot to replace de with nl.

**Comment by JonMagon** [comment]
Thank you for contribution 🙂

**Pull request merged**